rtl/include/eclhelper.hpp

```diff
@@ -435,6 +435,7 @@ struct RtlTypeInfo : public RtlITypeInfo
     inline bool isSigned() const { return (fieldType & RFTMunsigned) == 0; }
     inline bool isUnsigned() const { return (fieldType & RFTMunsigned) != 0; }
     inline bool isBlob() const { return getType() == type_blob; }
+    inline bool hasIfBlocks() const { return fieldType & RFTMinifblock; }
```

Comment:
@ghalliday  is this correct? The caller is calling this for actualFormat , RFTMinifblock (in ifblock) makes it sound like it's asking if this record is within a IFBLOCK, whereas we want to know whether this record structure contains IFBLOCKS

Reply:
sounds like this is wrong, will close PR for now